DOC-2045: Updated node upgrade page for CE 4.6 to 4.8 (#476)
corda/corda-docs: content/en/docs/corda-enterprise/4.7/node-upgrade-notes.md

Copy the PostgreSQL JDBC Driver *42.2.8* version *JDBC 4.2* into the `drivers` directory.

### 3.2. Extract DDL and DML script using Database Management Tool

To run the tool, use the following command:

```shell
java -jar tools-database-manager-<release number>.jar dry-run -b path_to_configuration_directory --core-schemas --app-schemas
```

The option `-b` points to the base directory (which contains a `node.conf` file, and `drivers` and `cordapps` subdirectories).

`--core-schemas` is required to adopt the changes made in the new version of Corda, and `--app-schemas` is related to the CorDapps changes.

A script named `migrationYYYYMMDDHHMMSS.sql` containing DDL and DML statements will be generated in the current directory.
This script will contain all the statements required to modify and create data structures (for example, tables/indexes),
and updates the Liquibase management table *DATABASECHANGELOG*.
The command doesn’t alter any tables itself.

{{< note >}}

> If your DDL and DML statements are run separately - DB admin runs DDL and an app user runs DML, you need to manually separate DDL and DML statements into separate scripts.

Reviewer: ```suggestion
If your DDL and DML statements are run separately (for example, if the DB admin runs DDL and an app user runs DML) you need to manually separate the DDL and DML statements into separate scripts.
```

### 3.4. Optional: Apply data updates on the H2 database

{{< note >}}

> You do not need to perform this step for databases other than H2 database.

Reviewer: ```suggestion
You only need to perform this step for the H2 database.
```
